mint-005.md

```diff
@@ -0,0 +1,165 @@
+| Scenario 18       |      |      | ✅    |       | ✅     | ✅     | ✅                                   |
+
+#### Layer 3:
+
+| Emergency Recovery Path | $PAK_1$ | $PAK_2$ | $PAK_3$ | $SAK$ | BIP-113 greater than `between_epoch_timestamp` |
+|:-----------------------:|:-------:|:-------:|:-------:|:-------:|:-----------------------------------------------:|
+| 2 of 3 PAKs AND SAK AND after BIP-113 time is greater than `between_epoch_timestamp`  | ![PAK1](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![PAK2](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![PAK3](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![SAK1](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![Timelock](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/unlock.png) |
+
+
+##### Valid Layer 3 Spend Conditions
+
+| Spending Scenario | PAK_1 | PAK_2 | PAK_3 | SAK | BIP-113 time greater than  (between_epoch_timestamp) |
+|-------------------|:-----:|:-----:|:-----:|:---:|:----------------------------------:|
+| Scenario 19       | ✅     | ✅     |       | ✅  | ✅                                  |
+| Scenario 20       | ✅     |       | ✅     | ✅  | ✅                                  |
+| Scenario 21       |       | ✅     | ✅     | ✅  | ✅                                  |                                |
+
+
+#### Layer 4:
+| Sovereign Recovery Path | $RK_1$ | $RK_2$ | $RK_3$ | Network BIP-113 time greater than `largest_epoch_timestamp` |
+|:-----------------------:|:------:|:------:|:------:|:-------:|
+| 2 of 3 RKs AND after BIP-113 time is greater than `largest_epoch_timestamp` | ![RK1](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![RK2](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![RK3](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![Timelock](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/unlock.png) |
+
+##### Valid Layer 4 Spend Conditions
+
+| Spending Scenario | RK_1 | RK_2 | RK_3 | Timelock (largest_epoch_timestamp) |
+|-------------------|:----:|:----:|:----:|:----------------------------------:|
+| Scenario 22       | ✅    | ✅    |      | ✅                                  |
+| Scenario 23       | ✅    |      | ✅    | ✅                                  |
+| Scenario 24       |      | ✅    | ✅    | ✅                                  |
+
+
+---
+
+# Example Miniscript Output Descriptor
+
+For this example, the `smallest_epoch_timestamp` is: 1672531200 (Jan 1 2023, midnight gmt), the `between_epoch_timestamp` is: 1673740800 and `largest_epoch_timestamp` is: 1675209600 (Feb 1 2023, midnight gmt)
+
+- MINT-005 Output Descriptor: 
```

Comment:
In 7bebe7efd503756903ccb9efc65e1e681a5ebb4e:

trailing whitespace

Reply:
Updated